### PR TITLE
implement cov

### PR DESCRIPTION
**Pull request by Hokchris**
https://github.com/unifyai/ivy/issues/6269

closes https://github.com/unifyai/ivy/issues/6269

Implemented cov for linear algebra section.

### CODE REVIEW

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hi @Hokchris!

Thanks for contributing to Ivy :slightly_smiling_face: !
Kindly add all these files in the experimental folder as it's related to the extension task. :slightly_smiling_face: 

Do let me know if you have any other questions!
Thanks! :slightly_smiling_face:

**Comment by Hokchris** [comment]
Hey @zaeemansari70 

Great, will do! Just checking that I should put them in all the respective experimental folders in each section (i.e. ivy/functional/backends/.../experimental/... .py for the backends, ivy_tests/test_ivy/test_functional/test_experimental/... for the tests, and so on)? And I currently have all backends except paddle implemented, guessing I should do an implementation of that too while I'm at it?

Thanks!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by sherry30** [comment]
Hi @Hokchris,

Thats correct, you should put them in their respective experimental folder.
You can try to do paddle backend as well if its possible, otherwise its fine without it.

**Comment by christianhon-reason** [comment]
Hey @sherry30 

Great, thanks! I'll try get it done by end of day today, if not I'll finish it tomorrow.

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py [hunk 1]
```diff
@@ -591,3 +591,140 @@ def test_cond(
...
+            small_abs_safety_factor=1,
+        )
+    )
+
+    return [dtype], value1, value2, rowVar, bias, ddof, fweights, aweights
+
+
+# cov
+@handle_test(
+    fn_tree="functional.ivy.cov",
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py:687]: can you change this to `functional.ivy.experimental.cov`

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

Your PR looks good, there is just that 1 change I pointed out.
Once you do that I'll merge your PR 😄 

Thanks

**Review by sherry30 — APPROVED** [review]
Everything looks good.
No worries about the paddle backend.
Thanks for the contribution 😄

**Pull request merged**